test/tracker.jl

```diff
@@ -432,4 +446,19 @@ end
   @test back([1, 1]) == (32,)
 end
 
+@testset "PDMats" begin
+    B = rand(5, 5)
+    S = PDMat(I + B * B')
+    @test gradtest(A -> S / A, (5, 5))
+    gradient(A -> sum(S / A), rand(5, 5))
```

Comment:
What's this extra `gradient` call for?

Reply:
Oh that was just for some initial debugging. I removed the `gradient` calls.